SquareLine/gen.py

```diff
@@ -0,0 +1,232 @@
+from genericpath import isdir
+import sys
+import os
+import json
+import shutil
+from colorama import Fore
+from zipfile import ZipFile
+
+# Directory of all boards
+BOARDS_DIR = "boards/"
+# Other board files, which are same for all boards (there can be used placeholders)
+COMMON_DIR = "common/"
+# JSON file which describing the board (must be in each board directory)
+BOARD_GENFILE = "manifest.json"
+# Files, which are specific for each board. Will be copied from boards/BOARD/ directory
+BOARD_SPECIFIC_FILES = {"sdkconfig.defaults", "main/idf_component.yml"}
+# Generated output directory
+OUT_DIR = "espressif"
```

Comment:
Can we add this directory to `.gitignore`, so that git doesn't consider the generated content as untracked files?